### DISCUSSION TITLE
[2092850]十字军骑士 {DLC无独立清单}

### DISCUSSION THREAD

**Discussion by SQDOL** [pjy612/SteamManifestCache#617] [Share]
### Steam 游戏ID

2092850

### 密钥列表

```hash
2092851;3ebf7530285db2ba77a8e47b6d69d7d345a3485d1b9f70cf2980011b43e8646e
```

### 清单附件

[清单.zip](https://github.com/pjy612/SteamManifestCache/files/15104679/default.zip)

### 备注

code:278BA3E3B04E2028
email:1571464825@qq.com

**Comment by pjy612** [comment]
* 密钥错误 请确认密钥长度

**Reply by SQDOL** [reply]
已修正
